<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestHardware</class>
 <widget class="QWidget" name="TestHardware">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>602</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TestHardware</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#widget_background
{
    background-color: #FFFFFF;
}
QLabel#label_title
{
    min-height: 40px;
    font: 20px;
	color: #ffffff;
    background-color: #0aa8e3;
}

QComboBox
{
    border: 1px solid #B9B9B9;
    color: #4A4A4A;
    background-color: #E1E1E1;
}
QComboBox:disabled
{
    background-color: rgb(190, 190, 190);
    color: #4A4A4A;
}
QComboBox:hover
{
    border: 1px solid #00A2E8;
}
QComboBox:drop-down
{
    border: 0px;
}
QComboBox::down-arrow
{
    image: url(:/common/common/drop-down-black.png);
}
QComboBox QAbstractItemView
{
    outline: none;
    border: 1px solid #949494;
    background-color: #FFFFFF;
}
QComboBox QAbstractItemView::item
{
    border: none;
    padding-left: 20px;
    height: 30px;
    color: #4A4A4A;
}
QComboBox QAbstractItemView::item:selected:active
{
    color: #FFFFFF;
    background-color: #0AA9E3;
}

QLabel
{
    color: #4A4A4A;
}
QPushButton
{
    outline: 0px;
    border: 0px;
    border-radius: 3px;
    padding-left: 20px;
    padding-right: 20px;
    min-width: 80px;
    min-height: 30px;
    color: #FFFFFF;
    background-color: #707070;
}
QPushButton:disabled
{
    background-color: rgb(190, 190, 190);
    color: #4A4A4A;
}
QPushButton:hover
{
    background-color: #0aa8e3;
}
QPushButton:pressed
{
    background-color: #7b7b7b;
}
QPushButton::checked
{
    background-color: #2B9CC6;
}

QToolButton
{
    outline: 0px;
    border: 0px;
    border-radius: 3px;
    min-width: 50px;
    padding: 2px;
    color: #FFFFFF;
    background-color: #707070;
}
QToolButton:disabled
{
    background-color: rgb(190, 190, 190);
    color: #4A4A4A;
}
QToolButton:hover
{
    background-color: #0aa8e3;
}
QToolButton:pressed
{
    background-color: #7b7b7b;
}
QToolButton::checked
{
    background-color: #2B9CC6;
}

QCheckBox
{
    outline: 0px;
    color: #4A4A4A;
}
QCheckBox::indicator
{
    width: 16px;
    height: 16px;
}
QCheckBox::indicator:unchecked
{
    border-image: url(:/common/common/checkbox-unchecked.png);
}
QCheckBox::indicator:unchecked:hover
{
    border-image: url(:/common/common/checkbox-unchecked-hover.png);
}
QCheckBox::indicator:unchecked:disabled
{
    border-image: url(:/common/common/checkbox-unchecked-disable.png);
}
QCheckBox::indicator:checked
{
    border-image: url(:/common/common/checkbox-checked.png);
}
QCheckBox::indicator:checked:disabled
{
    border-image: url(:/common/common/checkbox-checked-disable.png);
}
QCheckBox::indicator:indeterminate
{
    border-image: url(:/common/common/checkbox-partiallychecked-hover.png);
}
QCheckBox::indicator:indeterminate:disabled
{
    border-image: url(:/common/common/checkbox-partiallychecked-disable.png);
}

QLineEdit
{
    background-color: #E1E1E1;
    color: #4A4A4A;
    border: 1px solid #b9b9b9;
}
QLineEdit:disabled
{
    background-color: rgb(190, 190, 190);
    color: #4A4A4A;
}
QLineEdit:read-only
{
    background-color: rgb(190, 190, 190);
}
QLineEdit:hover
{
    border: 1px solid #1D9AFF;
}

QGroupBox
{
    outline: none;
}
QGroupBox::title
{
    color: #4A4A4A;
}
QGroupBox::indicator
{
    width: 16px;
    height: 16px;
}
QGroupBox::indicator:unchecked
{
    border-image: url(:/common/common/checkbox-unchecked.png);
}
QGroupBox::indicator:unchecked:hover
{
    border-image: url(:/common/common/checkbox-unchecked-hover.png);
}
QGroupBox::indicator:unchecked:disabled
{
    border-image: url(:/common/common/checkbox-unchecked-disable.png);
}
QGroupBox::indicator:checked
{
    border-image: url(:/common/common/checkbox-checked.png);
}
QGroupBox::indicator:checked:disabled
{
    border-image: url(:/common/common/checkbox-checked-disable.png);
}
QGroupBox::indicator:indeterminate
{
    border-image: url(:/common/common/checkbox-partiallychecked.png);
}
QGroupBox::indicator:indeterminate:hover
{
    border-image: url(:/common/common/checkbox-partiallychecked-hover.png);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_background" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_title">
        <property name="text">
         <string>硬件接口测试</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QScrollArea" name="scrollArea">
             <property name="minimumSize">
              <size>
               <width>350</width>
               <height>0</height>
              </size>
             </property>
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>368</width>
                <height>679</height>
               </rect>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QGroupBox" name="groupBoxInfoMation">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="title">
                  <string>设备信息</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_7">
                  <item>
                   <layout class="QGridLayout" name="gridLayout">
                    <item row="0" column="1" rowspan="2">
                     <widget class="QLineEdit" name="lineEditHardware">
                      <property name="readOnly">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="1">
                     <widget class="QLineEdit" name="lineEditSoftware">
                      <property name="readOnly">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="0">
                     <widget class="QLabel" name="labelMAC">
                      <property name="text">
                       <string>MAC:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="1">
                     <widget class="QLineEdit" name="lineEditMAC">
                      <property name="readOnly">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                    <item row="4" column="0">
                     <widget class="QLabel" name="labelSN">
                      <property name="text">
                       <string>SN:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="4" column="1">
                     <widget class="QLineEdit" name="lineEditSN">
                      <property name="readOnly">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="0">
                     <widget class="QLabel" name="labelSoftware">
                      <property name="text">
                       <string>固件版本:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="0" rowspan="2">
                     <widget class="QLabel" name="labelHardware">
                      <property name="text">
                       <string>硬件版本:</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="groupBoxMonitor">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="title">
                  <string>主屏信息</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_8">
                  <item>
                   <widget class="QWidget" name="widget" native="true">
                    <layout class="QGridLayout" name="gridLayout_4">
                     <property name="leftMargin">
                      <number>0</number>
                     </property>
                     <property name="topMargin">
                      <number>0</number>
                     </property>
                     <property name="rightMargin">
                      <number>0</number>
                     </property>
                     <property name="bottomMargin">
                      <number>0</number>
                     </property>
                     <item row="0" column="1">
                      <widget class="QLabel" name="labelMonitorInfo1">
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                         <horstretch>2</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="text">
                        <string>TextLabel</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="0">
                      <widget class="QLabel" name="labelMonitor1">
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                         <horstretch>1</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="text">
                        <string>主屏信息</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_5">
                    <item>
                     <spacer name="horizontalSpacer_4">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pushButtonMonitorUpdate">
                      <property name="text">
                       <string>更新</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <spacer name="horizontalSpacer_5">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="widgetTest" native="true">
                 <layout class="QVBoxLayout" name="verticalLayout_2">
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QGroupBox" name="groupBoxSettings">
                    <property name="title">
                     <string>VIDEO配置：</string>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_4">
                     <item>
                      <layout class="QGridLayout" name="gridLayout_2">
                       <item row="1" column="0">
                        <widget class="QLabel" name="label_2">
                         <property name="text">
                          <string>User:</string>
                         </property>
                        </widget>
                       </item>
                       <item row="2" column="0">
                        <widget class="QLabel" name="label_3">
                         <property name="text">
                          <string>Password:</string>
                         </property>
                        </widget>
                       </item>
                       <item row="0" column="1">
                        <widget class="QLineEdit" name="lineEditIP">
                         <property name="text">
                          <string>192.168.5.190</string>
                         </property>
                        </widget>
                       </item>
                       <item row="2" column="1">
                        <widget class="QLineEdit" name="lineEditPassword">
                         <property name="text">
                          <string>1</string>
                         </property>
                        </widget>
                       </item>
                       <item row="0" column="0">
                        <widget class="QLabel" name="label">
                         <property name="text">
                          <string>IP:</string>
                         </property>
                        </widget>
                       </item>
                       <item row="1" column="1">
                        <widget class="QLineEdit" name="lineEditUser">
                         <property name="text">
                          <string>admin</string>
                         </property>
                        </widget>
                       </item>
                       <item row="3" column="0">
                        <widget class="QLabel" name="label_4">
                         <property name="text">
                          <string>Video Codec:</string>
                         </property>
                        </widget>
                       </item>
                       <item row="3" column="1">
                        <widget class="QComboBox" name="comboBoxVideoCodec">
                         <item>
                          <property name="text">
                           <string>H.264</string>
                          </property>
                         </item>
                         <item>
                          <property name="text">
                           <string>H.265</string>
                          </property>
                         </item>
                        </widget>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QGroupBox" name="groupBoxTest">
                    <property name="title">
                     <string>测试项目：</string>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_5">
                     <item>
                      <widget class="QCheckBox" name="checkBoxAll">
                       <property name="text">
                        <string>全选</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <layout class="QVBoxLayout" name="verticalLayoutItems"/>
                     </item>
                     <item>
                      <layout class="QGridLayout" name="gridLayout_3">
                       <item row="0" column="0">
                        <widget class="QLabel" name="labelOutput">
                         <property name="text">
                          <string>HDMI/VGA输出</string>
                         </property>
                        </widget>
                       </item>
                       <item row="0" column="1">
                        <widget class="ComboBox" name="comboBoxOutput"/>
                       </item>
                       <item row="1" column="0">
                        <widget class="QLabel" name="labelScreen1">
                         <property name="text">
                          <string>HDMI1/VGA1分辨率</string>
                         </property>
                        </widget>
                       </item>
                       <item row="1" column="1">
                        <widget class="ComboBox" name="comboBoxScreen1"/>
                       </item>
                       <item row="2" column="0">
                        <widget class="QLabel" name="labelHDMI2">
                         <property name="text">
                          <string>启用HDMI2/VGA2</string>
                         </property>
                        </widget>
                       </item>
                       <item row="2" column="1">
                        <widget class="ComboBox" name="comboBoxHDMI2"/>
                       </item>
                       <item row="3" column="0">
                        <widget class="QLabel" name="labelScreen2">
                         <property name="text">
                          <string>HDMI2/VGA2分辨率</string>
                         </property>
                        </widget>
                       </item>
                       <item row="3" column="1">
                        <widget class="ComboBox" name="comboBoxScreen2"/>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_3">
                       <item>
                        <spacer name="horizontalSpacer_2">
                         <property name="orientation">
                          <enum>Qt::Horizontal</enum>
                         </property>
                         <property name="sizeHint" stdset="0">
                          <size>
                           <width>40</width>
                           <height>20</height>
                          </size>
                         </property>
                        </spacer>
                       </item>
                       <item>
                        <widget class="QPushButton" name="pushButtonStart">
                         <property name="text">
                          <string>开始</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QPushButton" name="pushButtonApply">
                         <property name="text">
                          <string>应用</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <spacer name="horizontalSpacer_3">
                         <property name="orientation">
                          <enum>Qt::Horizontal</enum>
                         </property>
                         <property name="sizeHint" stdset="0">
                          <size>
                           <width>40</width>
                           <height>20</height>
                          </size>
                         </property>
                        </spacer>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <spacer name="verticalSpacer">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="MyPlainTextEdit" name="plainTextEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>1</horstretch>
               <verstretch>1</verstretch>
              </sizepolicy>
             </property>
             <property name="undoRedoEnabled">
              <bool>false</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonClose">
             <property name="text">
              <string>退出</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyPlainTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>MyPlainTextEdit.h</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>combobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
